.gitignore

```diff
@@ -13,6 +13,8 @@
 *.egg
 .eggs/
 
+.venv/
```

Comment:
> If I'm right the .venv file is not generated with one of SCT script ? It's a manual action from you.

Yes, this was my understanding too.

@abelfodil I would recommend using the default conda environment created by SCT, rather than a virtualenv. You can find it at the location `$SCT_DIR/python/envs/venv_sct`, and you can easily activate the environment by running:

```console
joshua@tadpole:~$ source $SCT_DIR/python/bin/activate venv_sct
(venv_sct) joshua@tadpole:~$ 
```

Reply:
(And if this doesn't work for you, or you have other reasons for needing to use `.venv`, please let us know! I would love to understand better how you are developing for SCT.)

Reply:
I have multiple different workflows (depending on which machine I'm working on):
- Github codespaces
- Dev container
- Native manjaro
- Native macbook

For devcontainer and codespace I do not mind installing conda, but for any other workflow, a vanilla python venv works best for me.

Side note, would you be interested in a devcontainer PR for the project?

Reply:
Actually maybe we should even extend the scope of this PR and include common Python ignores from https://github.com/github/gitignore/blob/main/Python.gitignore to support more diverse workflows.

Reply:
> For devcontainer and codespace I do not mind installing conda, but for any other workflow, a vanilla python venv works best for me.

Just to clarify, you don't actually have to install `conda`. SCT brings its own copy of Miniconda and installs it to its source folder. So, it is just a matter of activating the already-present `venv_sct` using the already-present `conda`. :)

Reply:
One of the reasons I recommend working in the existing `venv_sct` environment is because it helps to ensure that any new dependencies that we explore are compatible with the existing dependencies required by SCT.

Reply:
Got it. Should we still add common ignores or just close this PR (either is fine by me)?